src/Entity/Mooc/Video.php

```diff
@@ -3,46 +3,16 @@
 namespace AppBundle\Entity\Mooc;
 
 use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
-use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\Common\Collections\Collection;
-use Gedmo\Mapping\Annotation as Gedmo;
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Component\Validator\Constraints as Assert;
-use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 
 /**
  * @ORM\Entity
- * @ORM\Table(
- *     uniqueConstraints={@ORM\UniqueConstraint(name="mooc_chapter_video_slug", columns="slug")}
- * )
- *
- * @UniqueEntity(fields={"slug", "chapter"})
  *
  * @Algolia\Index(autoIndex=false)
  */
 class Video extends BaseMoocElement
 {
-    /**
-     * @ORM\Column(type="smallint")
-     * @Gedmo\SortablePosition
-     */
-    protected $displayOrder;
-
-    /**
-     * @var Chapter
-     *
-     * @ORM\ManyToOne(targetEntity="Chapter", inversedBy="videos")
-     * @Gedmo\SortableGroup
-     */
-    protected $chapter;
-
-    /**
-     * @ORM\Column(length=800, nullable=true)
-     *
-     * @Assert\Length(min=5, max=800)
-     */
-    private $content;
```

Comment:
Where is this property ? can you place it in the `MoocBaseElement` ? It's needed also for the Quizz I think no ?
If it's needed only for the Video, can you undelete it ?

Reply:
Done, rename description to content